packages/medusa/src/utils/authenticate-middleware.ts

```diff
@@ -97,3 +98,12 @@ export const authenticate = (
     res.status(401).json({ message: "Unauthorized" })
   }
 }
+
+const getActorId = (authUser: AuthUserDTO, scope: string | RegExp) => {
+  if (stringEqualsOrRegexMatch(scope, "admin")) {
+    return authUser.app_metadata.user_id
+  } else if (stringEqualsOrRegexMatch(scope, "store")) {
+    return authUser.app_metadata.customer_id
+  }
+  return authUser.app_metadata.medusa_id
```

Comment:
What does this ID represent?

Reply:
rm'ed, no need to think about other scopes rn 😄